test/unittest/xmlparser/XMLLoadFileTests.cpp

```diff
@@ -0,0 +1,140 @@
+// Copyright 2020 Proyectos y Sistemas de Mantenimiento SL (eProsima
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include <thread>
+#include <mutex>
+#include <fstream>
```

Comment:
```suggestion

#include <mutex>
#include <fstream>
```

Unused header

Reply:
Done in e035ea991e1381ba57187339de12e6a3dc686370